build.gradle

```diff
@@ -209,7 +209,7 @@ task ktlint(type: JavaExec, group: "verification") {
 task printModules {
     doLast {
         subprojects.each { p ->
-            println "module: " + p.path
+            println "module: name=" + p.path + " buildPath=" + p.buildDir
```

Comment:
Maybe just make this JSON or YAML or something and make parsing really, really straightforward?

Reply:
+1

Another note: I love the fact we don't hardcode buildPaths anymore \o/

Reply:
My concern with using JSON/YAML is that it's not really consistent with the rest of the `gradle` output - there'd be a bunch of human-readable output, followed by a clump of JSON. I'd need to deliminate the JSON line, so that the importing tools wouldn't accidentally use the other `gradle` output as part of the JSON parse. Considering this, it seemed simpler and more elegant to follow the same style as the rest of the output

-----

Looking into this further, `gradle` has a `--quiet/-q` option which means that we can have `gradle` _only_ output our task's output. This means that executing `gradle -q printModulesJson` could strictly dump a block of machine-readable JSON, which is nice. I'll follow that approach